README.md

```diff
@@ -76,7 +76,7 @@ Running Enzyme Tests
 
 Enzyme is unopinionated regarding which test runner or assertion library you use, and should be
 compatible with all major test runners and assertion libraries out there. The documentation and
-examples for enzyme use [Mocha](https://mochajs.org/) and [Chai](http://chaijs.com/), but you
+examples for enzyme use [Mocha](https://mochajs.org) and [Chai](https://www.chaijs.com), but you
```

Comment:
```suggestion
examples for enzyme use [Mocha](https://mochajs.org) and [Chai](https://chaijs.com), but you
```

Reply:
I reverted this change via force-pushing the branch, as requested.